examples/mobile/HomeApp/js/app.js

```diff
@@ -1,11 +1,39 @@
 (function () {
 	"use strict";
-	var tau = window.tau;
+	var tau = window.tau,
+		state = {
+			widgets: {
+				news: false,
+				nowontv: false
+			}
+		};
+
+	function getWidgetsState() {
+		var latestNews = document.querySelector("#latest-news-check"),
+			nowOnTV = document.querySelector("#now-on-tv-check");
+
+		state.widgets.news = latestNews.checked;
+		state.widgets.nowontv = nowOnTV.checked;
+	}
+
+	function setWidgetsState() {
```

Comment:
How about changing the function name to the toggleWidgetsState()?
The implementation does not seem to match the function name.

Reply:
Done!